front/gatsby/src/apollo.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright 2018-2020 @paritytech/Nomidot authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { ApolloClient, InMemoryCache } from 'apollo-boost';
+import { split } from 'apollo-link';
+import { HttpLink } from 'apollo-link-http';
+import { WebSocketLink } from 'apollo-link-ws';
+import { getMainDefinition } from 'apollo-utilities';
+
+const httpLink = new HttpLink({
+  uri: 'https://34.76.106.193:4000',
+});
+
+const wsLink = new WebSocketLink({
+  uri: 'ws://34.76.106.193:4000',
```

Comment:
btw this should probably be hidden or access controlled somehow?

Reply:
@amaurymartiny bump

Reply:
For production, yeah for sure.

For now, probably a devops could answer you better, i'm not sure how okay it's to expose public endpoints on the test installation on gcp.

Reply:
cc: @fevo1971 @gabreal ?

Reply:
I guess I'm lacking some background here. is that portal going to be deployed on kubernetes? does it come with it's own polkadot client?

Reply:
Soo we have a server as a deployment on kubernetes and a loadbalancer service for clients (such as this frontend) to connect to. But right now anyone with the above ExternalIP is able to connect.

 It does not come with its own polkadot client at the moment, though that might be helpful.

Reply:
so there is no authentication? - as long as it is temporary it okayish but if you intend to use it as a development platform it's probably better to think about access restriction.